openapi3gen/openapi3gen_test.go

```diff
@@ -8,20 +8,6 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-type CyclicType0 struct {
-	CyclicField *CyclicType1 `json:"a"`
-}
-type CyclicType1 struct {
-	CyclicField *CyclicType0 `json:"b"`
-}
-
-func TestCyclic(t *testing.T) {
```

Comment:
Can you add a generator option that keeps the previous default behavior of returning an error? I'd like this capability to still be here, while changing to your work as the default.

Reply:
For sure

Reply:
For sure

Reply:
Option added and ready for review @fenollp